src/test/scala/uk/gov/hmrc/govukfrontend/views/components/GovukTabsSpec.scala

```diff
@@ -17,6 +17,52 @@
 package uk.gov.hmrc.govukfrontend.views
 package components
 
+import play.api.i18n.{Lang, Messages}
+import play.api.mvc.RequestHeader
+import play.api.test.FakeRequest
+import play.twirl.api.HtmlFormat
 import uk.gov.hmrc.govukfrontend.views.html.components._
+import uk.gov.hmrc.helpers.MessagesSupport
 
-class GovukTabsSpec extends TemplateUnitSpec[Tabs, GovukTabs]("govukTabs")
+import scala.util.Try
+
+class GovukTabsSpec extends TemplateUnitBaseSpec[Tabs]("govukTabs") with MessagesSupport {
+
+  private val component = app.injector.instanceOf[GovukTabs]
+
+  def render(templateParams: Tabs): Try[HtmlFormat.Appendable] = {
+    implicit val request: RequestHeader = FakeRequest("GET", "/foo")
+
+    Try(component(templateParams))
+  }
+
+  "GovukTabs" when {
+    "implicit messages language is English" should {
+      "display Egnlish default title if none passed in" in {
+        val tabs = Tabs(
+          title = ""
+        )
+
+        val content = component(tabs)
+
+        val button = content.select(".govuk-tabs__title")
+        button.text() shouldBe "Contents"
+      }
+    }
+
+    "implicit messages language is welsh" should {
+      val welshMessages: Messages = messagesApi.preferred(Seq(Lang("cy")))
+
+      "display welsh translation of default title if none passed in" in {
```

Comment:
capitalisation (sorry!)
```suggestion
    "implicit messages language is Welsh" should {
      val welshMessages: Messages = messagesApi.preferred(Seq(Lang("cy")))

      "display Welsh translation of default title if none passed in" in {
```